nautobot_floor_plan/forms.py

```diff
@@ -39,16 +50,59 @@ class Meta:
             "tile_depth",
             "x_axis_labels",
             "y_axis_labels",
+            "x_origin_seed",
+            "y_origin_seed",
```

Comment:
```suggestion
            "x_axis_labels",
            "x_origin_seed",
            "y_axis_labels",
            "y_origin_seed",
```

I think it would look better if these were grouped (X/X, Y/Y)